benchmark-service/benchmark/templates/index.html

```diff
@@ -1,23 +1,61 @@
-
-{% extends "base.html" %}
-{% block title %}
-Posts index
-{% endblock %}
+{% extends "layout.html" %}
+{% block title %} Benchmarks {% endblock %}
 
 {% block content %}
 <h1>Benchmark System</h1>
-<p>
-<ul>
-    {% for post in posts %}
-    <li>
-        <a href='/{{ post.id }}'>{{ post.title }}</a> {{ post.editor }}
-    </li>
-    {% endfor %}
-</ul>
-</p>
-<hr>
-<p>
-    <a href='/goose'>Goose</a>
-</p>
+
+<form action="{{ base_path }}/lookup" method="get">
```

Comment:
You can remove the `_csrf` field here because it's no longer a post command. Let's also change the URL to `{{ base_path }}/`